src/main/java/nextstep/subway/exception/CanNotFoundShortestPathException.java

```diff
@@ -0,0 +1,7 @@
+package nextstep.subway.exception;
+
+public class CanNotFoundShortestPathException extends RuntimeException{
```

Comment:
대괄호와 문자 사이에는 한 칸의 스페이스를 두는 것이 컨벤션입니다.

```suggestion
public class CanNotFoundShortestPathException extends RuntimeException {

Reply:
수정했습니다.